pkg/process/checks/process.go

```diff
@@ -702,3 +715,20 @@ func isDisallowListed(cmdline []string, disallowList []*regexp.Regexp) bool {
 	}
 	return false
 }
+
+func isECSFargatePidModeSetToTask(totalProcs int, containers []*model.Container) bool {
+	if !fargate.IsFargateInstance() {
+		return false
+	}
+
+	// aws-fargate-pause container only exists when "pidMode"" is set to "task" on ecs fargate
+	pidModeTaskContainerExists := false
```

Comment:
[minor] You can just use the var style declaration as the default value is already false.
```suggestion
	var pidModeTaskContainerExists bool
```

Reply:
I feel explicitly setting the variable to false is more clear. However, should we prefer the var declaration?

Reply:
Check out the [zero values of variables](https://go.dev/tour/basics/12). 

From the [uber style guide](https://github.com/uber-go/guide/blob/master/style.md#local-variable-declarations)

Usually when initializing the variable with `:=` means you want something that is not the zero value.

Reply:
I definitely would agree with defining empty slices with the var format, but I don't feel fully convinced about the bool since I think it reads clearer. I did find some other references in the repo owned by us, but I can change it if we can decide as a team this is the standard we want to continue with. Thoughts?